scripts/jenkins/Makefile.jenkins

```diff
@@ -132,11 +132,12 @@ test-py-booklets:
 	cd h2o-docs/src/booklets/v2_2015/source && ../../../../../scripts/run.py --numclouds 1 --numnodes 3 --baseport 53534 --jvm.xmx 5g --test pybooklet.glm.vignette.py
 
 test-py-demos:
-	export NO_GCE_CHECK=True && export MPLBACKEND=Agg && cd h2o-py/demos/ && ../../scripts/run.py --wipeall --numclouds 4 --baseport 56789 --jvm.xmx 10g
-
-test-py3-only-demos:
+	test "$$CI" != "true" || pip install -U pandas==1.0.5 numpy==1.20.2
```

Comment:
Can we document why we upgrade pandas and numpy, which files should be affected?

Reply:
Done

Reply:
Thanks